features.html

```diff
@@ -63,30 +63,61 @@ <h2 class="subtitle">
                 </div>
             </div>
         </section>
-            <div class="intro column is-8 is-offset-2">
-                <h2 class="title">Trusted messaging.
-                    </h2><br>
-                <p class="subtitle">We are primarily focused upon trusted secure structed messaging in a peer to peer manner.</p>
-            </div>
             <div class="sandbox">
                 <div class="tile is-parent is-shady">
                     <article class="tile is-child notification is-white">
-                        <p class="title">Building lightweight self-signed certificates, instead of X.509</p>
-                        <p class="subtitle">Identify uniquely a user of a system or an application that is running in the system.</p>
+                        <p class="title">Overview of Enhanced Security Features</p>
+                        <p class="subtitle">The Black Lager project now allows you to do the following on top of the Meshtastic infrastructure.</p>
                         <div class="content">
-                            <p>The Black Lager network has public-private keys stored client-side to ensure communication is encrypted.
+                            <p>
+                                <ul>
+                                <li>Create lightweight public-key certificates and digital signatures with the Ed25519 algorithm.</li>
+                                <li>Generate a persona with a name and public/private key pair using the Python application.</li>
+                                <li>Send and verify signed text messages with owned and peer personas stored on the client-side.</li>
+                                <li>Digital signature scheme uses SHA-512 hashing and Curve25519 elliptic-curve cryptography.</li>
```

Comment:
This should say "Signed messages with the Ed25519 public-key signature system which uses uses Curve25519 elliptic-curve cryptography and SHA-512 hashing." Possibly combine this bullet point with the previous bullet point.

Reply:
Have added a dot point for this.